doc/content/htgr/triso/triso_model.md

```diff
@@ -84,12 +83,28 @@ opens between the buffer and IPyC layers.
 
 !listing htgr/triso_fuel/triso_1d.i block=gen language=cpp
 
+`TRISO2DMeshGenerator` creates a 2D mesh appropriate for use in TRISO analysis.  The user supplies radial coordinates that mark the boundaries of mesh blocks.  A list of numbers of elements per block is also supplied. A `0` for the elements in the block represents a gap and is typically used for the gap that opens between the buffer and IPyC layers. Aspherical meshes are supported, allowing the mesh to have a flat facet at the bottom of the particle.  This is accomplished through the `aspect_ratio` parameter. Two varieties of aspherical meshes are available.  These are selected through the `aspherical_type` parameter.  The first of these, `vary_buffer`, creates meshes with thinned buffers.  The resulting mesh includes a small fillet at the junction of the flat facet and the original, circular geometry.  The second type, `vary_outer`, thins the outer layers slightly near the centerline and thickens them toward the edge of the facet.
+
+!listing htgr/triso_fuel/triso_2d_aspherical.i block=gen language=cpp
+
+`TRISO3DMeshGenerator` creates a 3D mesh appropriate for use in TRISO analysis.  The user supplies radial coordinates that mark the boundaries of mesh blocks.  A list of numbers of elements per block is also supplied. A `0` for the elements in the block represents a gap and is typically used for the gap that opens between the buffer and IPyC layers.
+
+!listing htgr/triso_fuel/triso_3d.i block=gen language=cpp
+
 ## UserObjects
```

Comment:
the descriptions in these sections need to be specialized a little for the 1D 2D and 3D models.

when the objects are exactly the same (kernels for example), just mention that?
when there's anything different, let s mention it

then what about the new Contact block?